docs/.vuepress/config.js

```diff
@@ -92,6 +92,7 @@ module.exports = {
 
           '/build/': [
             'get-started',
+            'hosted-lotus-node',
```

Comment:
By _hosted_ do you mean running a Lotus node on a remote server somewhere?

Reply:
You can either run this lotus node on your local machine or a remote server. It will server the same purpose